snapshotter/cmd/cc-snapshotter/main.go

```diff
@@ -0,0 +1,177 @@
+//go:build linux
+// +build linux
+
+/*
+   Copyright The containerd Authors.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"net"
+
+	"google.golang.org/grpc"
+
+	snapshotsapi "github.com/containerd/containerd/api/services/snapshots/v1"
+	"github.com/containerd/containerd/contrib/snapshotservice"
+
+	"github.com/urfave/cli"
+
+	"cc-snapshotter/snapshotter"
+)
+
+const usage = "cc-snapshotter stores image layers as ext4 filesystems with dmverity information"
+
+func main() {
+	app := cli.NewApp()
+	app.Version = "v0.1"
+	app.Name = "cc-snapshotter"
+	app.Commands = []cli.Command {
+		cli.Command {
+			Name: "run",
+			Usage: "run snapshotter using given config",
+			Flags: []cli.Flag {
+				cli.StringFlag {
+					Name: "config,c",
+					Usage:"Optional: Path to TOML configuration file",
+					Required: false,
+				},
+			},
+			Action: actionRunSnapshotter,
+		},
+		cli.Command {
+			Name: "dir2tar",
+			Usage: "convert directory to reproducible tar",
+			Flags: []cli.Flag {
+				cli.StringFlag {
+					Name: "dir,d",
+					Usage:"Required: Path to directory",
+					Required: true,
+				},
+				cli.StringFlag {
+					Name: "tar,t",
+					Usage:"Required: Path to tar file",
+					Required: true,
+				},
+			},
+			Action: actionDirectoryToTar,
+		},
+		cli.Command {
+			Name: "tar2disk",
```

Comment:
It seems that some of these commands are standard (like ``run`` and ``pull``) but are some command proprietary, like ``tar2disk``?